### ISSUE TITLE
Timezone issue

### ISSUE THREAD

**Issue by Split7fire** [flathub/im.riot.Riot#32]
Testing riot app from flathub on Ubuntu 18.04 & Fedora 29, timezone in all messages remain GMT. Any hints?

**Comment by SISheogorath** [comment]
Nope, can't reproduce on my Fedora 29 (Gnome). What Desktop environment are you running?

**Comment by Split7fire** [comment]
@SISheogorath I'm experience this on Fedora 29 (Gnome) and Ubuntu 18.04 (Budgie).

**Comment by measlytwerp** [comment]
Same issue on Arch Linux, all times in the interface are incorrect.

**Comment by SISheogorath** [comment]
Currently my times seem correct. @Split7fire can you verify this with the current stable version?

**Comment by Split7fire** [comment]
@SISheogorath I haven't got riot installed on my pc for now. What was the problem? Can you triage? Because some other apps on electron (example: Wire) still suffer from this problem.

**Comment by SISheogorath** [comment]
When I got it correctly, then while Riot is open the timestamps in conversations don't fit the local time.

Which when I checked yesterday, was the case. This might be due to the runtime upgrade to 19.08, but I don't really know, that's why I'm asking to reproduce it. I know that I tried before and sometimes it appeared (noticed during my holiday) and sometimes it doesn't (as I couldn't reproduce it originally).
